### ISSUE TITLE
Spiderman 2

### ISSUE THREAD

**Issue by Extra-1** [CHEINSTTROARLY/OWO#8]
Spider-Man 2 Script
Peter Parker

- She looks at me every day.

Mary Jane Watson.

Oh, boy. If she only knew

how I felt about her.

But she can never know.

Peter Parker

- I made a choice once to live

a life of responsibility.

A life she can never be a part of.

Peter Parker

- Who am I?

I'm Spider-Man, given a job to do.

And I'm Peter Parker,

and I, too, have a job.

Mr. Aziz

- Parker. Parker!

No, no, no, stop! Stop!

Parker, you're late, man. Always late.

Peter Parker

- I'm sorry, Mr. Aziz.

There was a disturbance.

Mr. Aziz

- Another disturbance.

Always a disturbance with you.

Come on, 21 minutes ago,

in comes order.

Harmattan, Burton & Smith.

Eight extra-large deep-dish pizzas.

In eight minutes, I am defaulting on

Joe's 29-minute guarantee.

Then, not only am I receiving

no money for these pizzas...

...but I will lose the customer forever

to Pizza Yurt.

Look, you are my only hope, all right?

You have to make it in time.

Peter, you're a nice guy.

But you're just not dependable.

This is your last chance.

You have to go 42 blocks

in 7 and 1/2 minutes or your ass is fired.

Mr. Aziz

- Go!

Car Driver

- Hey, what, are you stupid?

New Yorker

- Whoa! He stole that guy's pizzas!

Kids

- I'm gonna get it!

Spider-Man

- Hey, you guys.

No playing in the streets.

Kids

- Yes, Mr. Spider-Man.

Spider-Man

- See you.

New Yorker

- Way to go, Spidey!

Peter Parker

- Pizza time.

Receptionist

- You're late.

I'm not paying for those.

Mr. Aziz

- Joe's 30-minute guarantee

is a promise, man.

I know to you, Parker, a promise

means nothing. But to me, it's serious.

Peter Parker

- It's serious to me too, Mr. Aziz.

Mr. Aziz

- You're fired. Go.

Peter Parker

- Please, I need this job.

Mr. Aziz

- You're fired.

Peter Parker

- Look, give me another chance.

J. Jonah Jameson

- You're fired.

Parker, hello. You're fired.

Peter Parker

- Why?

J. Jonah Jameson

- Dogs catching Frisbees?

Pigeons in the park?

A couple geezers playing chess?

Betty Brant

- Boss.

J. Jonah Jameson

- Not now.

Peter Parker

- The Bugle could show another side

of New York for a change.

Robbie Robertson

- We got six minutes to deadline, Jonah.

We need page one.

J. Jonah Jameson

- I don't pay you to be a sensitive artiste.

I pay you... Still not now!

I pay you because that psycho

Spider-Man will pose for you.

Peter Parker

- He won't let me take any more pictures.

You turned the whole city against him.

J. Jonah Jameson

- A fact I'm very proud of.

Get your pretty little portfolio off my

desk before I go into a diabetic coma.

Betty Brant

- It's your wife, she lost her checkbook.

J. Jonah Jameson

- Thanks for the good news.

Peter Parker

- Please, isn't there any of these shots

you can use? I need the money.

J. Jonah Jameson

- Miss Brant. Get me a violin.

Robbie Robertson

- Five minutes to deadline, Jonah.

J. Jonah Jameson

- Run a picture of a rancid chicken.

Here's the headline:

"Food Poisoning Scare Sweeps City."

Ted Hoffman

- Some food got poisoned?

J. Jonah Jameson

- I'm a little nauseous.

Peter Parker

- All right, Mr. Jameson.

J. Jonah Jameson

- It stinks.

Robbie, there's your page one.

"Masked Menace Terrorizes Town."

Robbie Robertson

- I told you, he's not a menace.

J. Jonah Jameson

- I told you...

Robbie Robertson

- I'll take care of it.

J. Jonah Jameson

- I'll give you 150.

Peter Parker

- Three hundred.

J. Jonah Jameson

- That's outrageous.

Done. Give this to the girl.

Thank you. Bye-bye.

Peter Parker

- Hi.

Betty Brant

- Hey, Pete.

I don't think this covers the advance

I gave you a couple weeks ago.

Peter Parker

- Right.

Betty Brant

- Sorry.

Hey.

Chin up, okay?

Female Collage Student

- Watch it, jerk!

Peter Parker

- Dr. Connors. Sorry.

Curt Connors

- Where were you headed, Parker?

Peter Parker

- To your class.

Curt Connors

- My class is over.

See me standing here?

Peter Parker

- I'm sorry. I'm trying.

I wanna be here.

Curt Connors

- Then be here.

Look at you, Peter.

Your grades have been steadily

declining. You're late for class.

You always appear exhausted.

Your paper on fusion is still overdue.

Peter Parker

- I know. I'm planning to write it

on Dr. Otto Octavius.

Curt Connors

- Planning is not a major

at this university.

Octavius is a friend of mine.

Better do your research, Parker.

Get it done, or I'm failing you.

May Parker, Harry Osborn, Mary Jane Watson

- Surprise!

May Parker

- Well, say something.

Peter Parker

- What's the occasion?

May Parker

- Really, Peter. It's your birthday!

Whether you want

to remember it or not.

Mary Jane Watson

- He lives in another reality.

Don't you, Pete?

Peter Parker

- Hi, M.J.

Mary Jane Watson

- Hi.

Harry Osborn

- Hey, buddy.

Peter Parker

- Hey.

Mary Jane Watson

- Long time, no see.

Peter Parker

- So how's the play?

I read a great review.

Mary Jane Watson

- It's going fine. It's going good.

Harry Osborn

- She's brilliant in it.

Mary Jane Watson

- Harry sent me roses.

Harry Osborn

- So where you been, pal?

You don't return my calls.

Peter Parker

- I've been busy.

Harry Osborn

- Taking pictures of Spider-Man?

How's the bug these days?

May Parker

- The less you see of that man,

the better.

Now, let's all go into the other room

and have something to eat.

Mary Jane Watson

- I'll get the hors d'oeuvres.

Peter Parker

- So how are things going at Oscorp?

Harry Osborn

- They're great.

I'm head of Special Projects.

We're about to make

a breakthrough on fusion.

May Parker

- How lovely, Harry. Your father

would be so proud, rest his soul.

Harry Osborn

- Thank you.

We're actually funding

one of your idols, Pete. Otto Octavius.

Peter Parker

- I'm writing a paper on him.

Harry Osborn

- You want to meet him?

Peter Parker

- You'd introduce me?

Harry Osborn

- You bet. Octavius is

gonna put Oscorp on the map...

...in a way my father

never even dreamed of.

May Parker

- M.J., could you give me a hand?

Harry Osborn

- She's waiting for you, pal.

Peter Parker

- What do you mean?

Mary Jane Watson

- The way she looks at you

or doesn't look at you.

However you want to look at it.

Peter Parker

- I don't have time for girls right now.

Harry Osborn

- Why, are you dead?

Peter Parker

- I've been kind of busy.

Harry Osborn

- Taking pictures of your friend?

Peter Parker

- Could we get off that subject?

I want us to be friends, Harry.

I want us to trust each other.

Harry Osborn

- Then be honest with me.

If you knew who he was,

would you tell me?

Peter Parker

- May.

May Parker

- Oh, what, Ben?

Wait.

Oh, my.

Oh, Peter. Oh, for a second there,

I thought I was years ago.

Everybody's gone, aren't they?

Did they have a good time?

I'm sure they did.

Peter Parker

- You okay?

May Parker

- Of course. But you go home.

And be careful. I don't like that

scooter thing you drive around.

Peter Parker

- I'm worried about you.

You're so alone.

And I saw the letter from the bank.

May Parker

- Oh, my. You did?

Oh, well.

So?

I'm a little behind.

Everybody is.

Anyway, I don't want

to talk about it anymore.

I'm tired,

and you better start back home.

Here, kiddo. Happy birthday.

You need it more than I do.

Peter Parker

- No, I can't take that from you.

May Parker

- Yes, you can!

You can take this money from me.

For God's sake, it's not much.

Now, take it!

And don't you dare leave it here.

Oh, I'm sorry.

It's just that I miss

your uncle Ben so much.

Can you believe that it's two years

next month since he was taken?

I think to myself at times...

...were I to face the one responsible

for what happened, I'd...

Oh, I don't know what I'd do.

Now...

...you better take

the rest of your cake home.

Mary Jane Watson

- Hey.

Peter Parker

- Hey. You're still here.

I saw your billboard on Bleecker.

Mary Jane Watson

- Isn't it funny?

I'm really kind of embarrassed.

Peter Parker

- Don't be. It's nice.

I get to see you every day now.

Mary Jane Watson

- I liked seeing you tonight, Peter.

Peter Parker

- Oh, boy, yeah.

Mary Jane Watson

- "Oh, boy, yeah" what?

Peter Parker

- Nothing.

Mary Jane Watson

- Do you want to say something?

Peter Parker

- I...

...was...

...wondering if you're still

in the Village.

Mary Jane Watson

- You're such a mystery.

Peter.

What?

Happy birthday.

I'm seeing somebody now.

Peter Parker

- You mean, like a boyfriend?

Mary Jane Watson

- Well, like I like him.

What?

Peter Parker

- Nothing.

That's good, you know?

Companionship...

Mary Jane Watson

- May be more than that.

Peter Parker

- More?

Mary Jane Watson

- I don't know.

Peter Parker

- I'm coming to see your play

tomorrow night.

Mary Jane Watson

- You're coming?

Peter Parker

- I'll be there.

Mary Jane Watson

- Don't disappoint me.

Peter Parker

- I won't.

Mr. Ditkovitch

- Rent.

Peter Parker

- Hi.

Mr. Ditkovitch

- Hi. What's "hi"?

Can I spend it?

Peter Parker

- I have a paycheck due this week

and...

Mr. Ditkovitch

- You're a month late again. Again.

Peter Parker

- I promise as soon as...

Mr. Ditkovitch

- If promises were crackers,

my daughter would be fat.

Peter Parker

- I'm really sorry, Mr. Ditkovitch. All I got

is this 20 for the rest of the week.

Mr. Ditkovitch

- "Sorry" doesn't pay the rent.

And don't try to sneak past me. I have

ears like a cat and eyes like a rodent.

Peter Parker

- Thanks, Mr. Ditkovitch.

Ursula

- Hi, Pete.

Peter Parker

- Hello?

Mr. Ditkovitch

- Rent?

Raymond

- Doctor? Mr. Osborn's here.

Harry Osborn

- Nobel Prize, Otto. Nobel Prize.

And we'll all be rich.

Dr. Otto Ocatvius

- It's not about the prizes, Harry.

Harry Osborn

- But you need money.

You need Oscorp.

Dr. Otto Ocatvius

- Who do we have here?

Harry Osborn

- This is my good friend

I called you about.

He got me through

high-school science.

Peter Parker

- Peter Parker, sir.

I'm writing a paper on you for...

Dr. Otto Ocatvius

- I know what you're doing here...

...but I don't have time

to talk to students now.

But Oscorp pays the bills, so...

Harry Osborn

- That's why I have to take off.

Board meeting.

But my job is done here.

Got you two geniuses together.

Good luck tomorrow, Otto.

Nobel Prize. We'll see you in Sweden!

Dr. Otto Ocatvius

- Interesting fellow, your friend.

Peter Parker

- I won't take much of your time.

Dr. Otto Ocatvius

- Now I remember you. You're Connors'

student. He tells me you're brilliant.

He also tells me you're lazy.

Peter Parker

- I'm trying to do better.

Dr. Otto Ocatvius

- Being brilliant's not enough,

young man. You have to work hard.

Intelligence is not a privilege,

it's a gift.

And you use it

for the good of mankind.

You want to try it back there?

Peter Parker

- So is that it?

Dr. Otto Ocatvius

- Yes. My design to initiate

and sustain fusion.

Peter Parker

- I understand you use harmonics

of atomic frequencies.

Sympathetic frequencies.

Harmonic reinforcement?

Dr. Otto Ocatvius

- Go on.

Peter Parker

- An exponential increase

in energy output.

Dr. Otto Ocatvius

- A huge amount of energy.

Like a perpetual sun providing

renewable power for the whole world.

Peter Parker

- Are you sure you could stabilize

the fusion reaction?

Dr. Otto Ocatvius

- Peter, what have we been talking

about for the last hour and a half?

This is my life's work.

I certainly know the consequences

of the slightest miscalculation.

Peter Parker

- I'm sorry. I didn't mean

to question you.

Dr. Otto Ocatvius

- Rosie, our new friend thinks

I'm gonna blow up the city.

You can sleep soundly tonight.

Rosie Octavius

- Otto's done his homework.

Come to the demonstration tomorrow,

and you'll see for yourself.

And you need

to sleep soundly tonight, Otto.

Dr. Otto Ocatvius

- Did Edison sleep

before he turned on the light?

Did Marconi sleep

before he turned on the radio?

Did Beethoven sleep

before he wrote the th?

Peter Parker

- Did Bernoulli sleep before he found

the curves of quickest descent?

Dr. Otto Ocatvius

- Rosie, I love this boy.

Rosie Octavius

- Peter, tell us about yourself.

Do you have a girlfriend?

Peter Parker

- Well...

I don't really know.

Dr. Otto Ocatvius

- Well, shouldn't you know?

I mean, who would know?

Rosie Octavius

- Leave him alone.

Maybe it's a secret love.

Dr. Otto Ocatvius

- Love should never be a secret.

If you keep something as complicated

as love stored up inside...

...gonna make you sick.

I finally got lucky in love.

Rosie Octavius

- We both did.

Dr. Otto Ocatvius

- But it's hardly perfect.

You have to work at it.

Rosie Octavius

- I met him on the college steps,

and I knew it wasn't going to be easy.

He was studying science,

and I was studying English literature.

Dr. Otto Ocatvius

- That's right. I was trying to explain

the theory of relativity.

And Rosie was trying

to explain T.S. Eliot.

I still don't understand

what he was talking about.

Rosie Octavius

- Yes, you do.

Dr. Otto Ocatvius

- I'm serious.

T.S. Eliot is more complicated

than advanced science.

But if you want to get a woman to fall

in love with you, feed her poetry.

Peter Parker

- Poetry.

Dr. Otto Ocatvius

- Never fails.

Peter Parker

- A tall and slender maiden

All alone upon a prairie

Brightest green were all her garments

And her hair was like the sunshine

Day by day he gazed upon her

Mr. Ditkovitch

- Rent!

Where is my money?!

Louise'

- You seem jittery tonight.

Mary Jane Watson

- You never know who's coming.

Producer

- Ladies, five minutes. Five minutes.

New York Kid

- How'd you do that?

Peter Parker

- Work out, plenty of rest.

You know, eat your green vegetables.

New York Kid

- That's what my mom is always saying.

I just never actually believed her.

Armed Thief

- Come on. Go.

Keep it steady.

Black Lady

- It's a web.

Go, Spidey, go!

Armed Thief

- We got trouble!

Get me down!

Mary Jane Watson

- I am more than content

with what Mr. Moncrieff said.

His voice alone inspires one

with absolute credulity.

Louise'

- Then you think we should

forgive them?

Mary Jane Watson

- Yes.

I mean, no.

Hey. Hey. Hey, chief.

Police Officer

- You park there, I'm towing it.

Peter Parker

- Whatever.

Ticket Controller

- Shoelace.

You might want to...

- Can I help you?

Peter Parker

- Yeah, I've come to see the show.

Ticket Controller

- Oh, I'm sorry, sir. No one will be

seated after the doors are closed.

It helps maintain the illusion.

Miss Watson, she's a friend of mine.

Peter Parker

- She asked me to come.

Ticket Controller

- But not to come late.

Peter Parker

- I have to see this show.

Just let me in, I'll stand in the...

Street Performer Chinese Lady

- A Spider-Man

A Spider-Man

A-does whatever a spider can

A-spins a web, any size

Catches thieves a-just like flies

Look out

Here comes the Spider-Man

Louise'

- You sure you don't wanna

come tonight?

Mary Jane Watson

- Yeah, I'm sure.

Louise'

- Okay. Call me.

Mary Jane Watson

- I will.

Louise'

- You were great tonight.

Mary Jane Watson

- Thanks.

John Jamenson

- May I have an autograph,

please, miss?

Mary Jane Watson

- What are you doing here?

John Jamenson

- You hungry?

Mary Jane Watson

- Starved.

Spider-Man

- What was that?

Okay.

Elevator Passenger

- Cool Spidey outfit.

Spider-Man

- Thanks.

Elevator Passenger

- Where'd you get it?

Spider-Man

- I made it.

Elevator Passenger

- looks uncomfortable.

Spider-Man

- Yeah, it gets kind of itchy.

And it rides up in the crotch

a little bit too.

Mary Jane Watson

- Hi. It's me.

Sing your song at the beep.

Peter Parker

- Hi, M.J.

This is Peter.

I was on my way to your show and...

Well, I was on my bike...

Are you there?

I really was planning on it all day.

And...

...I know you predicted

I'd disappoint you.

Mary Jane Watson

- Bingo.

Peter Parker

- It's amazing, isn't it?

How complicated a simple thing...

...like being someplace at :

can become.

Actually, there was

this obnoxious usher.

Somebody has to talk

to that usher, M.J.

Call Recorder

- Your time has expired.

Please deposit cents

for the next five minutes.

Peter Parker

- I wanna tell you the truth.

Here it is.

I'm Spider-Man.

Weird, huh?

Now you know

why I can't be with you.

If my enemies found out about you...

...if you got hurt,

I could never forgive myself.

I wish I could tell you

how I feel about you.

Dr. Otto Ocatvius

- Ladies and gentlemen...

...my wife Rosie and I would like to

welcome you this afternoon.

But first, before we start...

...has anybody lost a large roll

of -dollar bills in a rubber band?

Because we found the rubber band.

It's a terrible joke.

But thank you for coming.

Today...

...you will witness the birth

of a new fusion-based energy source.

Safe, renewable energy

and cheap electricity for everyone.

And now let me introduce

my assistants.

These four actuators were

developed and programmed...

...for the sole purpose of creating

successful fusion.

They are impervious to heat

and magnetism.

These smart arms are controlled

by my brain through a neural link.

Nanowires feed directly

into my cerebellum...

...allowing me to use these arms

to control fusion reaction...

...in an environment

no human hand could enter.

Science Reporter

- Doctor, if the artificial intelligence

in the arms...

...is as advanced as you suggest...

...couldn't that make you

vulnerable to them?

Dr. Otto Ocatvius

- How right you are.

Which is why I developed

this inhibitor chip...

...to protect

my higher brain function.

It means I maintain control of these

arms, instead of them controlling me.

And now on to the main event.

Give me the blue light, Rosie.

Precious tritium is the fuel

that makes this project go.

There's only 25 pounds of it

on the whole planet.

I'd like to thank Harry Osborn and

Oscorp Industries for providing it.

Harry Osborn

- Happy to pay the bills, Otto.

Dr. Otto Ocatvius

- Ladies and gentlemen...

...fasten your seat belts.

Raymond

- Doctor, we have

a successful fusion reaction.

Oscorp Old Worker

This is a breakthrough

beyond your father's dreams.

Harry Osborn

- Thank you.

Raymond

- We're producing

a thousand-megawatt surplus.

Dr. Otto Ocatvius

- The power of the sun...

...in the palm of my hand.

Keep calm! It's only a spike!

lt'll soon stabilize!

Woman Science Reporter

Ladies and gentlemen,

please clear the room.

Louise'

- We have a containment breach!

Rosie Octavius

- Otto, please get back!

Harry Osborn

- Shut it off, Otto! Shut it off!

Dr. Otto Ocatvius

- It will stabilize! It's under control!

Harry Osborn

- I'm in charge here!

It's my money! I'm in con...

This doesn't change anything.

Dr. Otto Ocatvius

- What are you doing?

Spider-Man

- Pulling the plug.

Dr. Otto Ocatvius

- No!

Rosie!

Fire Fighter

- Watch it.

That was too close.

Old Science Reporter

If he had more than a drop of tritium,

he could have destroyed the city.

Harry Osborn

- I'm ruined.

I have nothing left,

except Spider-Man.

Oscorp Advisor

- He saved your life, sir.

Harry Osborn

- He humiliated me by touching me.

Oscorp Advisor

- The press will be here soon.

I suggest we move on.

Harry Osborn

- What was he doing here anyway?

Doctor

- Molten metal penetrated

the spinal cavity...

...and fused the vertebrae

at multiple points...

...including the lamina

and the roof of the spinal column.

We won't know the extent of

the damage until we get in there.

I suggest we cut off these mechanical

arms, slice up the harness...

...and, if need be,

consider a laminectomy...

...with posterior spinal fusion

from C -T to T .

Nurse

- We're ready, doctor.

Doctor

- Anybody here take shop class?

Nurse

- Help me! Help me!

Dr. Otto Ocatvius

- No!

J. Jonah Jameson

- It's all over town, Robbie.

Gossip. Rumors.

Panic in the streets, if we're lucky.

Crazy scientist turns himself

into some kind of a monster.

Four mechanical arms welded

right onto his body.

Guy named Otto Octavius winds up

with eight limbs. What are the odds?

Ted Hoffman

- Hoffman!

J. Jonah Jameson

- What are we gonna call this guy?

Ted Hoffman

- "Dr. Octopus."

J. Jonah Jameson

- That's crap.

Ted Hoffman

- "Science Squid"?

J. Jonah Jameson

- Crap!

Ted Hoffman

- "Dr. Strange."

J. Jonah Jameson

- That's pretty good.

But it's taken.

Wait, wait! I got it. "Dr. Octopus."

Ted Hoffman

- I like it.

J. Jonah Jameson

- Of course you do.

Dr. Octopus. New villain in town.

"Doc Ock."

Ted Hoffman

- Genius.

J. Jonah Jameson

- What, are you looking for a raise?

Get out.

Betty Brant

- Chief, I found Parker.

J. Jonah Jameson

- Where you been?

Why don't you pay your phone bill.

Mad scientist goes berserk,

we don't have pictures!

Robbie Robertson

- I heard Spider-Man was there.

J. Jonah Jameson

- Where were you?

Photographing squirrels? You're fired.

Betty Brant

- Chief, the planetarium party.

J. Jonah Jameson

- You're un-fired.

I need you, come here.

What do you know

about high society?

Peter Parker

- Oh. Well, l...

J. Jonah Jameson

- Don't answer that.

My society photographer got hit in the

head by a polo ball. You're all I got.

Big party for an American hero.

My son, the astronaut.

Peter Parker

- Could you pay me in advance?

J. Jonah Jameson

- You serious? Pay you for what?

Standing there?

The planetarium, tomorrow night, 8:00. There's the door.

Dr. Otto Ocatvius

- My Rosie's dead.

My dream is dead.

And these...

...monstrous things should be

at the bottom of the river...

...along with me.

Something...

...in my head.

Something talking.

The inhibitor chip!

Gone.

Rebuild.

No.

Peter was right.

I miscalculated.

I couldn't have miscalculated.

It was working, wasn't it?

Yes.

We could rebuild.

Enlarge the containment field.

Make it bigger

and stronger than ever.

But we need money.

Steal it? No, no, no, I'm not a criminal.

That's right.

The real crime would be not to finish

what we started.

We'll do it here.

The power of the sun

in the palm of my hand.

Nothing will stand in our way.

Nothing!

May Parker

- That's the social security.

Bank Teller

- Yes, I see.

Peter Parker

- My uncle Ben's life insurance.

Bank Teller

- Yes, but I'm afraid...

...it's just not sufficient

to refinance your home.

May Parker

- Oh, but I'm giving

piano lessons again.

Peter Parker

- You are?

Bank Teller

- We appreciate

that you've just opened up...

...a new supersaver account

with us today.

But the fact is...

...you do not have the assets

to justify this loan. I'm sorry.

May Parker

- Well...

At least we get the toaster.

Bank Teller

- Actually, that's only

with a deposit of or more.

May Parker

- Oh, yes, I see.

Bank Teller

- Okay.

Peter Parker

- Don't worry, we'll figure it out.

May Parker

- Peter?

Don't leave me.

Bank Teller

- Oh, that boy of yours is a real hero.

Security Guards

- Hold it!

Put your arms up! All of them!

Spider-Man

- Here's your change!

Oh, no. Come on!

Dr. Otto Ocatvius

- You're getting on my nerves.

Spider-Man

- I have a knack for that.

Dr. Otto Ocatvius

- Not anymore.

Police Agents

- Freeze! Don't move!

- Freeze!

- Hold your fire!

Dr. Otto Ocatvius

- Don't follow me.

Office Lady

- Wait a second.

Spider-Man

- Hand her over.

Dr. Otto Ocatvius

- Of course.

Spider-Man

Easy, now.

Dr. Otto Ocatvius

- Butterfingers.

Stan Lee

- Look out!

May Parker

- Help me!

- Help me!

Spider-Man

- I'm coming!

Hang on!

May Parker

- Help!

Thank you.

Spider-Man

- Aunt May.

Dr. Otto Ocatvius

- You've stuck your webs

in my business for the last time.

Now you'll have this woman's death

on your conscience.

Come on.

May Parker

- Shame on you.

Spider-Man

- There you go.

May Parker

- Thank you.

Oh, have I been wrong about you.

Spider-Man

- We sure showed him.

May Parker

- What do you mean, "we"?

Spiderman Simps

- Take me with you, Spider-Man.

- Take me.

Peter Parker

- May I?

High Society Old Lady

- Sure. Go ahead.

High Society Old Man

- Penny! Over here!

Harry Osborn

- Leave it.

Peter Parker

- Hey. Might wanna take it easy, buddy.

Harry Osborn

- Why? It's a party.

Wouldn't you be drinking if you lost

a bundle on some crackpot...

...who you thought was gonna take

you with him to fame and fortune?

Not to mention your friend the bug.

Peter Parker

- Not tonight, Harry.

Harry Osborn

- Every night!

Until I find him, it's 24-7.

J. Jonah Jameson

- Parker!

Parker!

Parker! What, are you deaf?

I called you twice.

Think I'm paying you

to sip champagne?

Get a shot of my wife

with the minister.

Beautiful tie.

Get a shot of us with the DA.

Beautiful dress.

Here, get a shot of the mayor

and his girlfriend.

Wife.

Ladies and gentlemen,

good evening.

High Society Woman Announcer

- The committee for

the Science Library of New York...

...is pleased to present

our guest of honor.

He's the first man to play football

on the moon.

The handsome, the heroic,

the delicious...

...Captain John Jameson.

Peter Parker

- Hi.

Mary Jane Watson

- You.

Peter Parker

- Listen, I'm sorry.

But there was a disturbance.

Mary Jane Watson

- I don't know you.

And I can't keep thinking about you.

It's too painful.

Peter Parker

- I've been reading poetry lately.

Mary Jane Watson

- Whatever that means.

Peter Parker

- Day by day he gazed upon her

Day by day he sighed with passion

Day by day...

Mary Jane Watson

- Don't start.

Peter Parker

- Can I get you a drink?

Mary Jane Watson

- I'm with John. He'll get me my drink.

Peter Parker

- John.

Mary Jane Watson

- By the way, John has seen

my show five times.

Harry has seen it twice.

Aunt May has seen it.

My sick mother got out of bed to see it.

Even my father.

He came backstage to borrow cash.

But my best friend,

who cares so much about me...

...can't make an 8:00 curtain.

After all these years...

...he's nothing to me

but an empty seat.

Harry Osborn

- It pisses me off, your loyalty to

Spider-Man and not to your best friend.

I find him with my father's body,

and you defend him.

Peter Parker

- Take it easy.

Harry Osborn

- Don't push me.

Don't act like you're my friend.

You stole M.J. from me.

You stole my father's love.

Then you let him die because you

didn't turn in the freak. Isn't that right?

Huh? Isn't that right?

Huh, brother?

John Jamenson

- Ladies and gentlemen,

I just want you all to know...

...that the beautiful Miss Mary Jane

Watson has just agreed to marry me.

High Society Party Guest

- What a surprise.

J. Jonah Jameson

- Parker, wake up! Wake up!

Shoot the picture!

Spider-Man

- Oh, no. No!

Why is this happening to me?

Dr. Davis

- You seem very okay to me.

My diagnosis?

It's up here.

You say you can't sleep.

Heartbreak? Bad dreams?

Peter Parker

- There is one dream where...

...in my dream...

...l'm Spider-Man.

But I'm losing my powers.

I'm climbing a wall, but I keep falling.

Dr. Davis

- So you're Spider-Man.

Peter Parker

- In my dream.

Actually, it's not even my dream.

It's a friend of mine's dream.

Dr. Davis

- Somebody else's dream.

What about this friend?

Why does he climb these walls?

What does he think of himself?

Peter Parker

- That's the problem.

He doesn't know what to think.

Dr. Davis

- Gotta make you mad not to know

who you are. Your soul disappears.

Nothing as bad as uncertainty.

Listen...

...maybe you're not supposed to be

Spider-Man climbing those walls.

That's why you keep falling.

You always have a choice, Peter.

Peter Parker

- I have a choice.

Ben Parker

- All the things you've

been thinking about, Peter...

...make me sad.

Peter Parker

- Can't you understand?

I'm in love with Mary Jane.

Ben Parker

- Peter, all the times

we've talked of honesty...

...fairness, justice...

Out of those times,

I counted on you to have the courage...

...to take those dreams

out into the world.

Peter Parker

- I can't live your dreams anymore.

I want a life of my own.

Ben Parker

- You've been given a gift, Peter.

With great power

comes great responsibility.

Take my hand, son.

Peter Parker

- No, Uncle Ben.

I'm just Peter Parker.

I'm Spider-Man...

...no more.

No more.

No more.

New Yorker hit by Tyre

- You punk!

Peter Parker

- Sorry.

Thanks.

Hot Dog Vendor

- You're welcome.

<'big>Police Officers===

- Hurry up!

Curt Connors

- And when the borderline is tau

equals zero, the eigenvalues are...?

Someone, please?

Peter Parker

- .23 electron volts.

Curt Connors

- Excellent work today, Parker.

Keep it up.

Male Theater Opera Lead

- I'm not really wicked at all.

You mustn't think that I'm wicked.

Mary Jane Watson

- If you are not, then you have

certainly been deceiving us all...

...in a very inexcusable manner.

I hope you have not been

leading a double life...

...pretending to be wicked

and being really good all the time.

That would be hypocrisy.

Male Theater Opera Lead

- Of course, I have been

rather reckless.

Theater Producer

- "I am glad."

"I am glad."

"I am glad."

Male Theater Opera Lead

- Are you glad to hear it?

Mary Jane Watson

- I am glad...

...to hear it.

Male Theater Opera Lead

- In fact, now you mention

the subject...

...I have been very bad

in my own small way.

Peter Parker

- You were so wonderful.

That was such a great play.

Mary Jane Watson

- You could've told me

you were coming.

Peter Parker

- I was afraid you'd say,

"Don't come."

Mary Jane Watson

- You look different.

Peter Parker

- I shined my shoes, pressed my pants,

did my homework.

I do my homework now.

You wanna get some chow mein?

Mary Jane Watson

- Peter...

...l'm getting married.

Peter Parker

- I always imagined you

getting married on a hilltop.

Mary Jane Watson

- And who's the groom?

Peter Parker

- You hadn't decided yet.

Mary Jane Watson

- You think just because

you saw my play...

...you can talk me out

of getting married?

Peter Parker

- You once told me you loved me.

I let things get in the way before.

There was something I thought

I had to do. I don't have to.

Mary Jane Watson

- You're too late.

Peter Parker

- Will you think about it?

Mary Jane Watson

- Think about what?

Peter Parker

- Picking up where we left off.

Mary Jane Watson

- Where was that?

We never got on.

You can't get off if you don't get on.

Peter Parker

- I don't think it's that simple.

Mary Jane Watson

- Of course you don't,

because you complicate things.

Peter Parker

- You don't understand.

I'm not an empty seat anymore.

I'm different.

Punch me, I bleed.

Mary Jane Watson

- I have to go.

I'm getting married in a church.

You are different.

J. Jonah Jameson

- Dear, we agreed to put on a wedding,

not go into bankruptcy.

Caviar?

What, are we inviting the czar?

Get some cheese and crackers,

some of those little cocktail weenies.

Betty Brant

- I got a garbage man here says he's

got something you might want to see.

J. Jonah Jameson

- If you have an extraterrestrial's head in

there, you're the third guy this week.

Robbie Robertson

- Where the hell did you get that?

Garbage Man

- In the garbage.

J. Jonah Jameson

- In the garbage?

He must've given up.

Thrown in the towel.

Abandoned his sad little masquerade.

I finally got to him.

The power of the press triumphs.

Garbage Man

- Yeah, yeah, yeah. Now, look, I think

I deserve a little something for this.

- Give you 50 bucks.

Garbage Man

- I could get more than that on eBay.

J. Jonah Jameson

- All right, 100. Give this man his

money and throw in a bar of soap.

Betty Brant

- Your wife is still on that line.

J. Jonah Jameson

- Flowers? How much?

You spend any more on this thing,

you can pick the daisies off my grave.

Get plastic.

Gotta go, big story.

Street Performer Chinese Lady

- Spider-Man

Spider-Man

Where have you gone to

Spider-Man?

Where have you gone to

Spider-Man?

Street Thug

- Come on, man. Get the money.

Come on, come on.

Mugged New Yorker

- Help!

Street Thug

- Come on!

May Parker

- It wasn't fair to have gone like that.

He was a peaceful man.

And it was all my doing.

Peter Parker

- Aunt May.

You don't have to punish yourself.

May Parker

- Oh, I know I shouldn't.

It's just that you wanted

to take the subway...

...and he wanted to drive you.

If only I had stopped him.

We'd all three of us

be having tea together.

Peter Parker

- I'm responsible.

May Parker

- For what?

Peter Parker

- For what happened to Uncle Ben.

May Parker

- But you were at the library.

You were doing your homework.

Peter Parker

- He drove me to the library,

but I never went in.

May Parker

- What do you mean?

Peter Parker

- I went someplace else.

Someplace where I thought

I could win some money...

...to buy a car, because I wanted...

...to impress Mary Jane.

It happened so fast.

I won the money, and the guy

wouldn't pay me. Then he got robbed.

The thief...

...was running towards me.

I could've stopped him,

but I wanted...

...revenge.

I let him go.

I let him get away.

He wanted a car.

He tried to take Uncle Ben's.

Uncle Ben said no...

...and then he shot him.

Uncle Ben was killed that night...

...for being the only one

who did the right thing.

And I...

I held his hand...

...when he died.

I've tried to tell you so many times.

Dr. Otto Ocatvius

- Just one more little chore.

Harry Osborn

- Where are you?

Bernard Houseman

- I'm leaving for the night, sir.

Harry Osborn

- Fine.

Bernard Houseman

- Your father only obsessed

over his work.

Harry Osborn

- Good night, Bernard.

Dr. Otto Ocatvius

- Hello, Harry.

Harry Osborn

- Otto. What do you want?

Dr. Otto Ocatvius

- That precious tritium.

But I need more of it this time.

More tritium, are you crazy?

You'll destroy the city! You're a hack!

Hey! Hey!

Stop! Stop! All right!

All right, all right, all right.

Put me down. We'll make a deal.

Kill Spider-Man,

I'll give you all the tritium you need.

On second thought,

bring him to me alive.

Dr. Otto Ocatvius

- How do I find him?

Harry Osborn

- Peter Parker.

Dr. Otto Ocatvius

- Parker?

Harry Osborn

- He takes pictures of Spider-Man

for the Bugle.

Make him tell you where he is.

Dr. Otto Ocatvius

- Have it ready.

Harry Osborn

- Don't hurt Peter!

New Yorker

- Somebody, help!

Fire!

Alisa, the Chens can't find their

daughter. Have you seen her?

Somebody call the fire department!

Help!

- Help!

Peter Parker

- Is anybody in that building?

New Yorker

- We think there's a kid

stuck on the second floor.

Hey, where you going?!

Peter Parker

- I'm coming!

It's okay. It's okay. Come here.

I'm gonna get you out of here.

Come on. Come on.

Rescued Kid

- Mama.

Fire Fighter 1

- You got some guts, kid.

Fire Fighter 2

- Some poor soul got trapped

on the fourth floor. Never made it out.

Oh, man.

Fire Fighter 1

- All right, Billy, let's knock it down

and roll them up. Get on out of here.

Let's pick it up! Let's get all this

equipment, let's go! Let's move it!

Peter Parker

- Am I not supposed to have

what I want?

What I need?

What am I supposed to do?

Ursula

- I shouldn't have without...

...knocking.

Peter Parker

- Come in.

Ursula

- Hi.

Peter Parker

- Hi.

Ursula

- Would you like a piece

of chocolate cake?

Peter Parker

- Okay.

Ursula

- And a glass of milk?

Peter Parker

- That would be nice.

Ursula

- Okay.

Peter Parker

- Thank you

Ursula

- You got a message.

It's your aunt.

Peter Parker

- Thanks.

What's going on?

May Parker

- Oh, they gave me another few weeks,

but I decided the hell with it.

I'm moving on.

I found a small apartment.

Peter Parker

- Why didn't you tell me?

May Parker

- I'm quite able

to take care of things myself.

And Henry Jackson across

the street is giving me a hand...

...and I'm giving him $5.

Peter Parker

- That's Henry Jackson?

May Parker

- Yes. It's funny what happens

in two years.

Nine years old, has great ambitions.

Peter Parker

- Listen, about my last visit...

May Parker

- Pish-posh, we needn't talk about it.

It's water over the dam or under

the bridge or wherever you like it.

But you made a brave move

in telling me the truth.

And I'm proud of you.

And I thank you, and I...

I love you, Peter.

So very, very much.

Peter Parker

- Hey. Where are all my comic books?

May Parker

- Those dreadful things?

I gave those away.

Henry Jackson

- I put the pans in the box, Mrs. Parker.

May Parker

- Thank you, Henry.

Henry Jackson

- Hi, Peter.

Peter Parker

- Hey, Henry.

You're getting tall.

May Parker

- Henry, why don't you put those

cookbooks in with the mixer.

Henry Jackson

- Okay. You take

Spider-Man's pictures, right?

Peter Parker

- I used to.

Henry Jackson

- Where is he?

May Parker

- Henry and I agree. We don't see

his picture in the paper anymore.

Peter Parker

- He...

...quit.

Henry Jackson

- Why?

Peter Parker

- Wanted to try other things.

Henry Jackson

- He'll be back, right?

Peter Parker

- I don't know.

May Parker

- You'll never guess

who he wants to be.

Spider-Man.

Peter Parker

- Why?

May Parker

- He knows a hero when he sees one.

Too few characters out there,

flying around like that...

...saving old girls like me.

And Lord knows,

kids like Henry need a hero.

Courageous, self-sacrificing people...

...setting examples for all of us.

Everybody loves a hero.

People line up for them.

Cheer them. Scream their names.

And years later, they'll tell how

they stood in the rain for hours...

...just to get a glimpse of the one...

...who taught them to hold on

a second longer.

I believe there's a hero in all of us...

...that keeps us honest...

...gives us strength...

...makes us noble...

...and finally allows us to die

with pride.

Even though sometimes

we have to be steady...

...and give up the thing

we want the most.

Even our dreams.

Spider-Man did that for Henry...

...and he wonders where he's gone.

He needs him.

Do you think you could lift that desk

and put it into the garage for me?

But don't strain yourself.

Peter Parker

- Okay.

Strong focus on what I want.

I'm back!

I'm back!

My back.

My back.

John Jamenson

- Honey, you sure you don't want to

invite your friend, the photographer?

Mary Jane Watson

- Peter Parker?

Mary Jane Watson

- Positive.

John Jamenson

- Thought he was your pal.

Mary Jane Watson

- Peter Parker?

John Jamenson

- Yeah.

Mary Jane Watson

- He's just a great big jerk.

John Jamenson

- The world's full of great big jerks.

Mary Jane Watson

- You're adorable.

John Jamenson

- It's just the uniform.

Mary Jane Watson

- Lean your head back for me.

John Jamenson

- Put my head back?

Mary Jane Watson

- Yeah, just do it.

John Jamenson

- Just put my head back?

Okay.

Wow.

I'm back on the moon.

You up there with me?

Peter Parker

- Hiya.

Mary Jane Watson

- Surprised?

Peter Parker

- Very.

Thanks for coming

Peter Parker

- Everything okay?

Mary Jane Watson

- You might say so.

This is funny.

I'm not sure how to begin.

You know how our minds

play tricks on us.

Peter Parker

- Tell me about it.

Mary Jane Watson

- Well, mine did a real number on me.

Some part of me heard what you

had to say after the show that night...

...that you were different.

But I didn't want to listen.

I was afraid to trust you.

But I've been thinking about it...

Peter Parker

- Listen, there's more for me to say.

I maybe rushed into things.

I thought...

Mary Jane Watson

- Wait a minute.

What are you saying?

Peter Parker

- I'm saying...

I thought I could be there for you,

Mary Jane...

...but I can't.

My mind was playing tricks too.

Mary Jane Watson

- Do you love me, or not?

Peter Parker

- I...

...don't.

Mary Jane Watson

- You don't.

Kiss me.

Peter Parker

- Kiss you?

Mary Jane Watson

- I need to know something.

Just one kiss.

Mary Jane Watson

- What's happening?

Dr. Otto Ocatvius

- Peter Parker...

...and the girlfriend.

Peter Parker

- What do you want?

Dr. Otto Ocatvius

- I want you to find your friend

Spider-Man.

Tell him to meet me

at the Westside Tower at 3:00.

Peter Parker

- I don't know where he is.

Dr. Otto Ocatvius

- Find him.

Or I'll peel the flesh off her bones.

Peter Parker

- If you lay one finger on her...

Dr. Otto Ocatvius

- You'll do what?

Robbie Robertson

- Still no word on the whereabouts

of your son's fiancée.

Sorry, Jonah.

J. Jonah Jameson

- It's all my fault.

I drove Spider-Man away.

He was the only one who could've

stopped Octavius.

Ted Hoffman

- Yes.

J. Jonah Jameson

- Spider-Man was a hero.

I just couldn't see it.

He was a...

A thief! A criminal!

He stole my suit!

He's a menace to the entire city!

I want that wall-crawling

arachnid prosecuted!

I want him strung up by his web!

I want Spider-Man!

Spider-Man

- Where is she?

Dr. Otto Ocatvius

- Oh, she'll be just fine. Let's talk.

Train Passenger

- What the hell?

Dr. Otto Ocatvius

- You have a train to catch.

Train Conductor

- Help!

Help! Help! I can't stop it!

The brakes are gone!

Train Passenger

- It's Spider-Man.

Train Conductor

- Oh, my God, this is it!

Peter Parker

- Tell everyone to hang on!

Train Conductor

- Brace yourselves!

Any more bright ideas?

Peter Parker

- I got a few, yeah!

Train Conductor

- Whoa, watch out!

Train Passenger 2

- We're slowing down.

Train Passenger 3

- Nice and easy, we'll go down.

Slower. Gently.

Train Passenger 4

- Is he alive?

Train Passenger 5

- He's...

...just a kid.

No older than my son.

Train Kids

- It's all right.

We found something.

We won't tell nobody.

It's good to have you back,

Spider-Man.

Dr. Otto Ocatvius

- He's mine!

Train Passenger 1

- You want to get to him,

you got to go through me.

Train Passenger 2

- And me.

Train Passenger 3

- Me too.

Dr. Otto Ocatvius

- Very well.

Where do you want him?

And the tritium?

Yes.

Harry Osborn

- If only I could cause you the pain

that you've caused me.

First we'll see who's behind the mask.

I can look into your eyes as you die.

Pete.

No.

It can't be.

Peter Parker

- Harry.

Where is she?

Where is he keeping her?

- He's got M.J.

Harry Osborn

- No. All he wanted was the tritium.

Peter Parker

- Tritium?

He's making the machine again.

When that happens, she'll die,

along with half of New York.

- Now, where is he?

Harry Osborn

- Peter...

...you killed my father.

Peter Parker

- There are bigger things

happening here than me and you.

Harry, please, I've got to stop him.

Mary Jane Watson

- Hey!

Hey!

I'm talking to you!

You got what you needed for your little

science project, now let me go.

Dr. Otto Ocatvius

- I can't let you go,

you'd bring the police.

Not that anybody could stop me

now that Spider-Man's dead.

Mary Jane Watson

- He's not dead.

- I don't believe you.

Dr. Otto Ocatvius

- Believe it.

Spider-Man

- Surprise.

As soon as you get free...

Dr. Otto Ocatvius

- I should have known Osborn

wouldn't have the spine to finish you!

Spider-Man

- Shut it down, Ock. You're going

to hurt a lot more people this time.

Dr. Otto Ocatvius

- Well, that's a risk

we're willing to take.

Spider-Man

- Well, I'm not.

Hang on, Mary Jane!

Run!

Dr. Otto Ocatvius

- Let's see you scurry out of this.

Spider-Man

- Now what?

Peter Parker

- Dr. Octavius.

We have to shut it down.

Please tell me how.

Dr. Otto Ocatvius

- Peter Parker?

"Brilliant but lazy."

Peter Parker

- Look at what's happening.

We must destroy it.

Dr. Otto Ocatvius

- I can't destroy it.

I won't.

Peter Parker

- You once spoke to me

about intelligence.

That it was a gift to be used

for the good of mankind.

Dr. Otto Ocatvius

- A privilege.

Peter Parker

- These things have turned you

into something you're not.

Don't listen to them.

Dr. Otto Ocatvius

- It was my dream.

Peter Parker

- Sometimes, to do what's right,

we have to be steady...

...and give up the thing

we want the most.

Even our dreams.

Dr. Otto Ocatvius

- You're right.

He's right.

Listen.

Listen to me now.

Listen to me now.

Peter Parker

- Now, tell me how to stop it.

Dr. Otto Ocatvius

- It can't be stopped.

It's self-sustaining now.

Peter Parker

- Think!

Dr. Otto Ocatvius

- Unless...

The river.

Drown it.

I'll do it.

Peter Parker

- Noooooooooooooooooooooooooo!

Hi.

Mary Jane Watson

- Hi.

Peter Parker

- This is really heavy.

M.J.

In case we die...

Mary Jane Watson

- You do love me.

Peter Parker

- I do.

Mary Jane Watson

- Even though you said you didn't.

Dr. Otto Ocatvius

- I will not die a monster.

Mary Jane Watson

- I think I always knew...

...all this time...

...who you really were.

Peter Parker

- Then you know

why we can't be together.

Spider-Man will always have enemies.

I can't let you take that risk.

I will...

...always be Spider-Man.

You and I can never be.

John Jamenson

- Mary Jane!

Harry Osborn

- Hello?

Who's that?

Normann Osborn

- Son.

I'm here.

Harry Osborn

- Dad?

- I thought you were...

Normann Osborn

- No.

I'm alive in you, Harry.

Now it's your turn.

You swore to make Spider-Man pay.

Now make him pay.

Harry Osborn

- But Pete's my best friend.

Normann Osborsn

- And I'm your father.

You're weak.

You were always weak. You'll always

be weak until you take control.

Now you know the truth about Peter.

Be strong, Harry.

Avenge me.

Avenge me!

Harry Osborn

- No!

J. Jonah Jameson

- Call Deborah.

Mrs Jameson

- The caterer?

J. Jonah Jameson

- Tell her not to open the caviar.

Mary Jane Watson

- Had to do what I had to do.

Peter Parker

- Mary Jane.

Mary Jane Watson

- Peter.

I can't survive without you.

Peter Parker

- You shouldn't be here.

I know you think we can't be together.

Mary Jane Watson

- But can't you respect me enough

to let me make my own decision?

I know there'll be risks.

But I want to face them with you.

It's wrong that

we should only be half-alive...

...half of ourselves.

I love you.

So here I am,

standing in your doorway.

I've always been

standing in your doorway.

Isn't it about time somebody

saved your life?

Well, say something.

Peter Parker

- Thank you, Mary Jane Watson.

Mary Jane Watson

- Go get them, tiger.
